### ISSUE TITLE
Q_tau 初始化是无穷大

### ISSUE THREAD

**Issue by PeiqinSun** [z-hXu/ReCU#3]
你好, 很感谢你的repo代码贡献. 有几个问题请教一下:
` Q_tau = (- EW * torch.log(2-2*self.tau)).detach().cpu().item(), `
这个self.tau我看初始值好像给的是1, 这不 2-2=0吗, log之后不是无穷大吗? 论文有bug?

**Comment by z-hXu** [comment]
你好，model link中的config.txt文件中写有详细的训练超参数

**Comment by PeiqinSun** [comment]
谢谢回答. 我看到了tau有一个动态范围 [0.85, 0.99]; 那我想再请教一下, 训练代码中是会根据epoch动态更改tau吗?

**Comment by z-hXu** [comment]
是的，论文和代码中都有提到

**Comment by PeiqinSun** [comment]
恩, 我又细看了一下代码, 找到了. 实在不好意思, 再扰烦请教一下, 我看ReCU这个技巧只用在了conv-modules, fc不用吗? 默认最后一个fc是float?

**Comment by z-hXu** [comment]
是的，第一个卷积层和fc层计算消耗较小，而且这两层不做量化对BNN有很大收益，这是近几年BNN文章的一个共识，可以参考bi-real net
